### DISCUSSION TITLE
Audio on your pages

### DISCUSSION THREAD

**Discussion by SilentGlasses** [squidfunk/mkdocs-material#4149] [General]
Just wanted to post this short topic here in case anybody-else finds it useful. I had to drop an audio clip on a docs page, here is how I solved it..

You will need to have the [md_in_html](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown/?h=md_in_html#markdown-in-html) option to your config for this to work:

```
markdown_extensions:
  - md_in_html
```

Then add your audio blocks like so:

```
<audio controls="controls">
  <source type="audio/ogg" src="filename.ogg"></source>
</audio>
```

or

```
<audio controls="controls">
  <source type="audio/mp3" src="filename.mp3"></source>
</audio>
```

**Comment by facelessuser** [comment]
I have an experimental branch up for Pymdown Extensions.  It's something I'm not interesting in pushing without some real feedback though.

https://github.com/facelessuser/pymdown-extensions/pull/1465
